README.md

```diff
@@ -3,18 +3,38 @@
 ## Introduction
 This is a deck building app for Star Wars: The Card Game.  This project is inspired by (but not a fork of) the Alciende-created deck building sites (netrunnerdb.com, arkhamdb.com, ringsdb.com, etc).  This was lacking for the Star Wars game so I wanted to make one to help people get into this great game.
 
-## Hacking on this Project
-### You need
-- Ruby 3.2
-- Rails 7
-- Postgres 14
-
-### Getting it running
-- clone
-- configure your `database.yml` as needed
-- `bin/rails db:migrate` to set the db up
-- `bin/rails db:seed` to seed the DB with data from the cardXml directory (taken from OCTGN)
-- `bin/dev` to start it all up
+### Prerequisites
+
+1. For local deployment
+    - Ruby 3.2
+    - Rails 7
+    - Postgres 16
+    - Node.js 20
+2. For container deployment
+    - Docker
+    - Docker Compose
+
+### For Development
+1. `git clone https://github.com/kphurley/swlcgdb.git swlcgdb`
+2. `cd swlcgdb`
+3. Start postgres
+   4. Configure config/database.yml if required
+   5. To start a docker instance of postgres run `./postgres_docker.sh`
+4. `npx install esbuild`
```

Comment:
Small nit - This markdown renders a little oddly - mind cleaning that up?

It's probably ok here to have one long series of steps with (optional) in front of them.  I think the markdown renderer gets confused with this indentation.